src/core/public/injected_metadata/injected_metadata_service.ts

```diff
@@ -76,6 +76,7 @@ export interface InjectedMetadataParams {
         user?: Record<string, UserProvidedValues>;
       };
     };
+    logoURL: string;
```

Comment:
on line 65 i see vars do you think we could use that to include the values or should we just make it required? just thinking about future proofing like if we want to keep modify this params to add more branding this might get huge.

Reply:
That makes sense. But should we still keep all the branding related variables together, to not mix with other vars? Maybe we can make it a brandingVars: { [key: string] : unknown }, then two functions getBrandingVar() and getBrandingVars() like getInjectedVar() and getInjectedVars()?